docs (microservices): add documentation for KafkaContext.heartbeat usage (#2409)
nestjs/docs.nestjs.com: content/microservices/kafka.md

}
```

> If your endpoint involves slow processing time for each message you should consider calling `heartbeat` callback, use `getHeartbeat()` method of `KafkaContext` which will send heartbeat to the broker, as follows:

Reviewer: ```suggestion
If your endpoint involves slow processing time for each message you should consider using the `heartbeat` callback. To do this, use the `getHeartbeat()` method of the `KafkaContext` which will send heartbeat to the broker, as follows:
```

> If your endpoint involves slow processing time for each message you should consider using the `heartbeat` callback. To do this, use the `getHeartbeat()` method of the `KafkaContext` which will send heartbeat to the broker, as follows:

Reviewer 2: ```suggestion
If your handler involves a slow processing time for each received message you should consider using the `heartbeat` callback. To retrieve the `heartbeat` function, use the `getHeartbeat()` method of the `KafkaContext`, as follows:
```

```typescript
@@filename()
@MessagePattern('hero.kill.dragon')
async killDragon(@Payload() message: KillDragonMessage, @Ctx() context: KafkaContext) {

>   const originalMessage = context.getMessage();

Reviewer 2: ```suggestion
```

  const heartbeat = context.getHeartbeat();
  

>   // Do some slow processing:

Reviewer 2: ```suggestion
  // Do some slow processing
```

  await doWorkPart1();

>   // Send heartbeat when it's possible, in order to not exceed the sessionTimeout

Reviewer 2: ```suggestion
  // Send heartbeat to not exceed the sessionTimeout
```

  await heartbeat();

>   // Do some slow processing again:

Reviewer 2: ```suggestion
  // Do some slow processing again
```
